src/pages/lifetime/celebration/singlesday.tsx

```diff
@@ -0,0 +1,82 @@
+import Layout from '@/components/layout/Layout';
+import cookies from '@/lib/cookies';
+import TestimonialsSection from '@/components/home/TestimonialsSection';
+import Navbar from '@/components/layout/navbars/Navbar';
+import { MinimalFooter } from '@/components/layout/footers/MinimalFooter';
+import FAQSection from '@/components/shared/sections/FaqSection';
+import CtaSection from '@/components/shared/CtaSection';
+import PaymentSection from '@/components/lifetime/celebration/singles-day/PaymentSection';
+import FeatureSection from '@/components/lifetime/celebration/singles-day/FeatureSection';
+import HeroSection from '@/components/lifetime/celebration/singles-day/HeroSection';
+
+const SinglesdayCelebrationTemplate = ({
+  lang,
+  langJson,
+  navbarLang,
+  footerLang,
+  metatagsDescriptions,
+  textContent
+}) => {
+const metatags = metatagsDescriptions.filter((desc) => desc.id === 'singles-day');
+const discount=0.2;
+const percent='80%'
+const currencySpecified='US';
+//const couponCode=PromoCodeName.SinglesDay;
+
+  return (
+    <Layout
+      title={metatags[0].title}
+      description={metatags[0].description}
+      segmentName="SinglesDay"
+      lang={lang}
+      specialOffer={`https://internxt.com/images/previewLink/LifetimePreviewLink.png`}
+    >
+    <Navbar textContent={navbarLang} lang={lang} cta={['default']} fixed mode="payment" isLinksHidden hideNavbar />
+    <div className="mt-16">
+      <HeroSection textContent={langJson.HeroSection} percent={percent}/>
+      <PaymentSection
+        textContent={langJson.PaymentSection}
+        discount={discount}
+        showPriceBefore
+        lang={lang}
+        percent={percent}
+        currencySpecified={currencySpecified}
+        lifetimeMode="celebration"
+      />
+      <FeatureSection textContent={langJson.FeatureSection} backgroundColor='bg-white' />
```

Comment:
Maybe you can use the `TextAndCardGroupsColumnSecrtion` component directly. You have an example of how it works [here](https://github.com/internxt/website/blob/main/src/pages/affiliates/index.tsx#L205).